pkg/multiclusterengine/multiclusterengine.go

```diff
@@ -213,10 +212,10 @@ func GetCatalogSource(k8sClient client.Client) (types.NamespacedName, error) {
 		return nn, nil
 	}
 	if len(filtered) > 1 {
-		return nn, errors.New("Found more than one catalogSource with expected channel")
+		return nn, fmt.Errorf("Found more than one %s catalogSource with expected channel %s", DesiredPackage(), desiredChannel())
```

Comment:
Could you add the channel name we're looking for to this msg and the one below.  I think that will be helpful to customers or support folks tracking down issues.